src/components/input-date-picker/input-date-picker.stories.ts

```diff
@@ -83,3 +83,13 @@ export const darkModeRTL_TestOnly = (): string => html`
   </div>
 `;
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
+
+export const mediumIconForLargeInputDatePicker_TestOnly = (): string => html`
+  <calcite-input-date-picker
+    scale="l"
+    start="2020-12-12"
+    end="2020-12-16"
+    range=""
+    layout="horizontal"
+  ></calcite-input-date-picker>
+`;
```

Comment:
Instead of adding a new snapshot, can you add `scale="l"` to an existing one? When multiple things can be tested in a single snapshot without conflicting, that's ideal. E.g. the darkModeRTL stories.

I'd suggest adding it to `laoNumberingSystem_TestOnly` since they are testing completely separate parts of the UI